app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/Constants.kt

```diff
@@ -0,0 +1,4 @@
+package com.github.HumanLearning2021.HumanLearningApp
+
+
+const val EXTRA_LEARNING_MODE = "EXTRA_LEARNING_MODE"
```

Comment:
Isn't it best practice to give a name containing the package like ?
`const val EXTRA_MESSAGE = "com.example.myfirstapp.MESSAGE"`
https://developer.android.com/training/basics/firstapp/starting-activity.html

Also, I'm not sure that a Constants class is really a good idea. It will end up being used for all sorts of unrelated things. Better keep it in a companion object in the launching activity

Reply:
Didn't know about this, but yea sounds like a good idea.